### PR TITLE
Optimize themes and cleanup

### PR DESCRIPTION
**Pull request by jamie256**
* Update to beta build 5
* optimize
* beta feedback, and notebook updates

### CODE REVIEW

File: DESCRIPTION.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+- [Motivation](#motivation-)
+- [Features](#features-)
+- [Installation](#installation-)
+- [Usage](#usage-)
+- [Modules](#modules-)
+
+## Motivation 🎯
+
+Productionizing language models, including LLMs, comes with a range of risks due to the infinite amount of input combinations, which can elicit an infinite amount of outputs. The unstructured nature of text poses a challenge in the ML observability space - a challenge worth solving, since the lack of visibility on the model's behavior can have serious consequences.
```
Comment (richard-rogers) [inline DESCRIPTION.md:13]: Is productionizing a word? 🤣

File: DESCRIPTION.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+To install LangKit, use the Python Package Index (PyPI) as follows:
+
+```
+pip install langkit
+```
+
+## Usage 🚀
+
+LangKit modules contain UDFs that automatically wire into the collection of UDFs on String features provided by whylogs by default. All we have to do is import the LangKit modules and then instantiate a custom schema as shown in the example below.
```
Comment (richard-rogers) [inline DESCRIPTION.md:44]: Should we define UDFs, or do we expect the audience to already be familiar with it.

File: langkit/injections.py [hunk 3]
```diff
@@ -27,7 +27,7 @@ def init(model_path: Optional[str] = None):
     )
 
 
-@register_metric_udf(col_name=prompt)
+@register_metric_udf(col_name=_prompt)
```
Comment (richard-rogers) [inline langkit/injections.py:30]: Value error doesn't seem quite right here, though I'm not seeing a built in exception that fits better... maybe `LangKitException`?

File: langkit/sentiment.py [hunk 4]
```diff
@@ -5,16 +5,19 @@
 
 _lexicon = "vader_lexicon"
 _sentiment_analyzer = None
+_nltk_download_count = 0
```
Comment (richard-rogers) [inline langkit/sentiment.py:8]: boolean?

File: langkit/examples/Batch_to_Whylabs.ipynb [hunk 5]
```diff
@@ -48,9 +48,17 @@
   },
   {
    "cell_type": "code",
```
Comment (richard-rogers) [inline langkit/examples/Batch_to_Whylabs.ipynb:50]: line 46 s/containint/containing/

File: DESCRIPTION.md (hunk 1 above)
Comment (jamie256) [reply DESCRIPTION.md:13]: This is not my content, but I did cleanup some of it with Andre. I'm pulling in from the beta/examples branch.

File: DESCRIPTION.md (hunk 2 above)
Comment (jamie256) [reply DESCRIPTION.md:44]: Its a good call out. I'm hoping users will recognize them from places like pyspark.

File: langkit/injections.py (hunk 3 above)
Comment (jamie256) [reply langkit/injections.py:30]: Agreed. The Value error is more for debugging, whylogs will swallow these.

Let's do in a follow up.

Comment (jamie256) [reply langkit/injections.py:30]: https://github.com/whylabs/langkit/issues/55

File: langkit/examples/Batch_to_Whylabs.ipynb (hunk 5 above)
Comment (jamie256) [reply langkit/examples/Batch_to_Whylabs.ipynb:50]: Nice catch, fixing.

File: DESCRIPTION.md (hunk 2 above)
Comment (jamie256) [reply DESCRIPTION.md:44]: https://github.com/whylabs/langkit/issues/56

File: langkit/sentiment.py (hunk 4 above)
Comment (jamie256) [reply langkit/sentiment.py]: Sure, fixed.

File: langkit/examples/Intro_to_Langkit.ipynb [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langkit import llm_metrics # alternatively use 'light_metrics'\n",
+    "import whylogs as why\n",
+    "\n",
+    "why.init(session_type='whylabs_anonymous')\n",
```
Comment (richard-rogers) [inline langkit/examples/Intro_to_Langkit.ipynb:51]: Maybe explain `session_type` argument

File: langkit/examples/Intro_to_Langkit.ipynb [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+   ],
+   "source": [
+    "from langkit.whylogs.samples import load_chats, show_first_chat\n",
+    "\n",
+    "# Let's look at what's in this toy example:\n",
+    "chats = load_chats()\n",
+    "print(f\"There are {len(chats)} records in this toy example data, here's the first one:\")\n",
+    "show_first_chat(chats)\n",
+    "\n",
+    "results = why.log(chats, name=\"langkit-sample-chats-all\", schema=schema)"
```
Comment (richard-rogers) [inline langkit/examples/Intro_to_Langkit.ipynb:92]: Maybe explain `name` argument, allowed forms of first argument.

File: langkit/examples/Intro_to_Langkit.ipynb [hunk 8]
```diff
@@ -0,0 +1,185 @@
...
+    "You can explore and compare specific metrics, in this example we expect a large and obvious distribution drift in the sentiment scores on the response which you can see [here](https://hub.whylabsapp.com/resources/model-1/profiles?feature-highlight=response.sentiment_nltk&includeType=discrete&includeType=non-discrete&limit=30&offset=0&profile=ref-zGPRPgwvdjD0AiqB&profile=ref-1V0T1JdwlongNSf4&sessionToken=session-YgBnFz&sortModelBy=LatestAlert&sortModelDirection=DESC)."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Next Steps\n",
+    "If you see value in detecting changes in how your LLM application is behaving, you might take a look at some of our other examples showing how to monitor these metrics as a timeseries for an LLM application in production, or how to customize the metrics logged by using your own surrogate models or critic metrics.\n",
```
Comment (richard-rogers) [inline langkit/examples/Intro_to_Langkit.ipynb:154]: s/timeseries/time series/ ?

**Review by richard-rogers — APPROVED** [review]

**Pull request merged**